notebooks/lesson03_data-cleaning.ipynb

```diff
@@ -0,0 +1,1866 @@
+{
```

Comment:
<p>maybe you need to do the same thing for the postal codes and city numbers as well. however they are int/floats at the moment so probably you need to change the type first, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='91'/>

Reply:
<p>hmm true, but i don't want to convert <code>city</code>  and <code>postal_code</code>  to one-hot because then i'll have thousands of columns and that slows down the training step dramatically. i think i'll just convert the <code>postal_code</code>  to int and leave it as that</p>

Reply:
ah ok. then i'd leave a remark somewhere to clarify that these columns are also categorical but are not converted to one-hot for this reason.